katalog/forecastle/config.yaml

```diff
@@ -5,7 +5,7 @@
 ---
 namespaceSelector:
   any: true
-crdEnabled: false
+crdEnabled: true
```

Comment:
Does enabling the CRD disable the automatic detection using the ingress annotations or do both work simultaneously?

Reply:
both work simultaneously